kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/attempt-log-list.vue

```diff
@@ -1,44 +1,41 @@
 <template>
 
-  <div class="answer-history">
+  <div class="attempt-log-list">
```

Comment:
Just to follow up on this comment - it's not an absolute. Consistency of nomenclature across the system is also important, so there's always a balance to be achieved...